README.md

```diff
@@ -418,6 +419,16 @@ The Google Dapper paper describes how the Dapper tools allow them to associate a
 
 Arbitrary application-specific annotations could be a useful addition however, so this feature may be added in the future. Until then you can output log messages tagged with tracing and timestamp information to approximate the functionality.
 
+<a name="tags"></a>
+## Tags
+Tags allow for key-value pairs to be associated with a span as metadata, often useful for filtering or grouping trace information.  For example, say you've instrumented retry logic. It may be desireable to be able to distinguish between a span/trace that contains retries and those that didn't.  Or you may want to tag your spans based on an attribute of the request, like user type or an authenticated flag. 
+
+```
```

Comment:
in brave we say putTag to clarify replace semantics

Reply:
That's a good point. I'd prefer `putTag(...)`. Or just `tag(...)` with good javadocs explaining the semantics.

Reply:
I went with `putTag(..)` as I felt the semantics were most clear.